explorer/Explorer.tsx

```diff
@@ -597,6 +613,9 @@ export class Explorer
 
     render() {
```

Comment:
[componentDidUpdate](https://reactjs.org/docs/react-component.html#componentdidupdate) is probably a better option.

Reply:
Moved it there, and added links in the PR description to `halley` where it's staged.